javascript/compute-height.js

```diff
@@ -129,22 +128,21 @@ function reloadFrameElement(iframeElement) {
     iframeElement.src = oldSrc;
 }
 
-function forceFeedIdToIFrame(frameId) {
+function forceFeedIdToIFrame(workflowId) {

```

Comment:
I don't think we can rename this to `workflowId` because it's not the workflow that has an id (a "workflow" is a graph of nodes), it's the iframe itself. In my view, the iframeId targets a specific iframe

Reply:
My understanding is that there is a difference between workflows and worflow types, and it's important to see it in the code.

Reply:
Sure, I can rename the concept of workflows to workflow types as it overlaps with the existing concept of comfyui workflow. Would that make more sense in your view?